autonomy/cli/helpers/chain.py

```diff
@@ -72,47 +75,61 @@ def get_ledger_and_crypto_objects(
             f"RPC cannot be `None` for chain config; chain_type={chain_type}"
         )
 
-    if keys is None:
-        crypto = make_crypto("ethereum")
+    identifier = EthereumHWIApi.identifier if hwi else EthereumApi.identifier
```

Comment:
This is problematic. Please add a new PR to make it optional whether EthereumHWIApi is present. I.e. `open-aea-ledger-ethereum-hwi` should be an `extra` and not part of the base installation!